kernel/public/test-scenes/0.101.move-inside-scene/game.ts

```diff
@@ -0,0 +1,13 @@
+import { BoxShape, engine, Entity, log, moveTo, OnPointerUp, Transform, Vector3 } from 'decentraland-ecs/src'
+
+const e = new Entity()
+e.addComponent(new BoxShape())
+e.addComponent(new Transform({ position: new Vector3(4, 0, 6) }))
+e.addComponent(
+  new OnPointerUp((e) => {
+    log('clicked: ', e)
+    moveTo({ x: 10, y: 0, z: 10 }, { x: 4, y: 1, z: 6 })
```

Comment:
How about renaming the function to `movePlayerTo`, instead of `moveTo` ?

I think it would be clearer, otherwise people might wonder if that's to move an entity, not the player

Reply:
@nearnshaw yes, of course! that name is better. thanks